bin/create_release.sh

```diff
@@ -44,14 +44,15 @@ if [[ ! $bamboo_RELEASE_FLAG == true ]]; then
   exit 0
 fi
 
+# todo: The following check raises an error if the TAG exists, not a release. ORCA-606
 ## Check if the code has already been released.
-export url="https://github.com/nasa/cumulus-orca/releases/tag/"v$bamboo_ORCA_VERSION""
-if curl --output /null --silent --fail "$url"; then
-  echo "Release URL already exists: $url. Exiting."
-  exit 1
-else
-  echo "$url does not exist. Proceeding with release..."
-fi
+# export url="https://github.com/nasa/cumulus-orca/releases/tag/"v$bamboo_ORCA_VERSION""
+# if curl --output /null --silent --fail "$url"; then
+#   echo "Release URL already exists: $url. Exiting."
+#   exit 1
+# else
+#   echo "$url does not exist. Proceeding with release..."
+# fi
```

Comment:
How does this not check the release? If the release was made, then that url exists and curl returns true unless I am missing something.

Reply:
The check raises an error if the TAG exists, not a release.
The tag is created along with the release branch, so it will always exist at this point.

Reply:
I see. 
But for tag, the url is https://github.com/nasa/cumulus-orca/tags
For release, the url is https://github.com/nasa/cumulus-orca/releases

So the url in the script checks for the release and not the tag even if the tag exists

Reply:
Note that the full url is https://github.com/nasa/cumulus-orca/releases/tag/v7.0.0
And that page existed already, containing zips of the source code. Note that we do not push those zips manually, so it's likely an automagical thing.

Reply:
Interesting! Didn't expect this lol.